src/bedrock/common.cpp

```diff
@@ -0,0 +1,57 @@
+#include "communication.pb.h"
+#include "common.h"
+#include "threads.h"
+#include "requests.h"
+
+using namespace std;
+
+// node capacity in KB
+unsigned MEM_NODE_CAPACITY = 60000000;
```

Comment:
Why are these constants defined here whereas others are in the .h files?

Reply:
because they are global variables, not constants. You can't have global variables defined in .h files ... otherwise the second time you include that file it's a multiply-defined error.  If you prefer to make them constants I can do that and put them back in common.h; I originally didn't want to change any code, just reorg it.

Reply:
moved back to common.h and converted to constants